tests/installation/disk_activation_zfcp.pm

```diff
@@ -0,0 +1,73 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: This test module handles ZFCP disk activation
```

Comment:
Ok. Just spotted a disk_activation_iscsi and thought then its also good for zfcp...

Reply:
please, do not resolve the comments until you change the code, otherwise is hard to review or know what is changed. Normally I can see some 'oudated' code label in top to know that.

Reply:
also another internal rule we use is that, we should reply to all comments by if it is enough clear and you change the code you don't need to write a reply, just with the 'outdated' file message is understood.